src/components/attendee/ScheduleOverlay/ScheduleOverlay.tsx

```diff
@@ -202,11 +202,12 @@ export const ScheduleOverlay: React.FC = () => {
 
     const currentVenueBookMarkEvents = eventsFilledWithPriority.filter(
       ({ isSaved, spaceId: eventSpaceId }) =>
-        isSaved && eventSpaceId?.toLowerCase() === space?.id
+        isSaved && eventSpaceId?.toLowerCase() === space?.id.toLowerCase()
```

Comment:
[NOTE]

Useful utility function for the future `compareStringsNoCase( )` or similar that can be called every time we need to compare two strings and don't want to remember to check if `null`, `undefined` etc.